src/App.js

```diff
@@ -1,14 +1,20 @@
-import React from "react";
+import React,{Component} from "react";
 import Navigation from "./components/navbar";
 import SimpleFooter from "./components/footer";
+import TfVis from "./components/tf";
 
 
-export default function App() {
-  return (
-    <div>
-      <Navigation />
-      {/* <Home /> */
-      <SimpleFooter /> }
-    </div>
-  );
-}
\ No newline at end of file
+class App extends Component {
```

Comment:
keep this as function components, not class components  - keeping it constant across new React expectations